syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -462,6 +456,28 @@ def __getitem__(self, indices):
 
     ## SECTION SPDZ
 
+    def _convert_to_shared_tensor(self, other):
```

Comment:
what does this method (convert to shared tensor) exactly do? and what is "other"

Reply:
I removed this function and created a `to_shared_tensor` that only converts the tensor to shared_tensor 😃